[question] missing github.com/docker/libnetwork package in vendor

**Issue Description**
pull latest code, make install and fail

**Describe what happened**:

```
➜  pouch git:(master) make install
GOOS=linux go build -o pouchd
network/types/network.go:3:8: no Go files in /root/go/src/github.com/docker/libnetwork
daemon/mgr/network.go:17:2: cannot find package "github.com/docker/libnetwork/config" in any of:
        /root/go/src/github.com/alibaba/pouch/vendor/github.com/docker/libnetwork/config (vendor tree)
        /usr/local/go/src/github.com/docker/libnetwork/config (from $GOROOT)
        /root/go/src/github.com/docker/libnetwork/config (from $GOPATH)
        /root/go/src/github.com/docker/libnetwork/Godeps/_workspace/src/github.com/docker/libnetwork/config
network/bridge/bridge.go:11:2: cannot find package "github.com/docker/libnetwork/drivers/bridge" in any of:
        /root/go/src/github.com/alibaba/pouch/vendor/github.com/docker/libnetwork/drivers/bridge (vendor tree)
        /usr/local/go/src/github.com/docker/libnetwork/drivers/bridge (from $GOROOT)
        /root/go/src/github.com/docker/libnetwork/drivers/bridge (from $GOPATH)
        /root/go/src/github.com/docker/libnetwork/Godeps/_workspace/src/github.com/docker/libnetwork/drivers/bridge
daemon/mgr/network.go:18:2: cannot find package "github.com/docker/libnetwork/netlabel" in any of:
        /root/go/src/github.com/alibaba/pouch/vendor/github.com/docker/libnetwork/netlabel (vendor tree)
        /usr/local/go/src/github.com/docker/libnetwork/netlabel (from $GOROOT)
        /root/go/src/github.com/docker/libnetwork/netlabel (from $GOPATH)
        /root/go/src/github.com/docker/libnetwork/Godeps/_workspace/src/github.com/docker/libnetwork/netlabel
daemon/mgr/network.go:19:2: cannot find package "github.com/docker/libnetwork/options" in any of:
        /root/go/src/github.com/alibaba/pouch/vendor/github.com/docker/libnetwork/options (vendor tree)
        /usr/local/go/src/github.com/docker/libnetwork/options (from $GOROOT)
        /root/go/src/github.com/docker/libnetwork/options (from $GOPATH)
        /root/go/src/github.com/docker/libnetwork/Godeps/_workspace/src/github.com/docker/libnetwork/options
network/bridge/bridge.go:13:2: cannot find package "github.com/vishvananda/netlink" in any of:
        /root/go/src/github.com/alibaba/pouch/vendor/github.com/vishvananda/netlink (vendor tree)
        /usr/local/go/src/github.com/vishvananda/netlink (from $GOROOT)
        /root/go/src/github.com/vishvananda/netlink (from $GOPATH)
        /root/go/src/github.com/docker/libnetwork/Godeps/_workspace/src/github.com/vishvananda/netlink
Makefile:27: recipe for target 'server' failed
make: *** [server] Error 1
➜  pouch git:(master) git log --pretty=oneline | head -n 1
053ae8c19715fa3a3c5a6f55b99081c5fde08a7e Merge pull request #411 from Letty5411/test
```

**Describe what you expected to happen**:

it seems that we miss the github.com/docker/libnetwork package in vendor

**How to reproduce it (as minimally and precisely as possible)**:
pull code and make install

**Anything else we need to know?**:

ping @rudyfly and @allencloud 

**Environment**:
- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/Makefile
+++ b/Makefile
@@ -18,6 +18,7 @@ build: pre server client
 
 .PHONY: pre
 pre:
+	@ git submodule update --init
 	@ mkdir -p $(ORIG_GOPATH)/src/github.com/docker
 	@ - [ -L $(ORIG_GOPATH)/src/github.com/docker/libnetwork ] && rm -f $(ORIG_GOPATH)/src/github.com/docker/libnetwork
 	@ - ln -s $(shell pwd)/extra/libnetwork $(ORIG_GOPATH)/src/github.com/docker/libnetwork
@@ -38,7 +39,7 @@ clean:
 	./module --clean
 
 .PHONY: check
-check: fmt lint vet validate-swagger
+check: pre fmt lint vet validate-swagger
 
 .PHONY: fmt
 fmt: ## run go fmt

